src/main/java/frc/trigon/robot/misc/objectdetectioncamera/ObjectDetectionCamera.java

```diff
@@ -8,7 +9,9 @@ public class ObjectDetectionCamera extends SubsystemBase {
     private final ObjectDetectionCameraInputsAutoLogged objectDetectionCameraInputs = new ObjectDetectionCameraInputsAutoLogged();
     private final ObjectDetectionCameraIO objectDetectionCameraIO;
     private final String hostname;
-    private double lastVisibleObjectYaw = 0;
+    private double lastVisibleObjectYawDegrees = 0;
```

Comment:
I don't see the need for this to be a class variable, nor a valid explanation in my head to what this variable is.

Reply:
From what I understand, to track an object, it keeps track of it's last known yaw and continues tracking it by finding the target yaw with the least deviation from the previous known yaw.